src/UI/InputOptionsController.java

```diff
@@ -19,11 +19,25 @@ public String toString()
       return super.toString().replace("_", " ");
     }
   }
+  /**
+   *  Compare with {@link CombatContext.CalcType}
+   */
+  public static enum InputCalcType
+  {
+    NO_LUCK, PESSIMISTIC, OPTIMISTIC;
```

Comment:
Huh. It seems odd to make this user-configurable instead of providing the range. Yes, that's harder UI, so barring that, the `NO_LUCK` answer is probably the right one, especially since you can't change this setting in-game.

Admittedly, the "just let users configure what they want, it doesn't hurt anything" argument sounds compelling, but that can quickly turn into having too much user-configurable stuff that nobody (or even worse: a vocal 5% of users) actually uses.

Reply:
Wargroove was criticized by a fair number of people for having the displayed damage not be the minimum, but the middle of the range. That's probably mostly coming from people expecting AW things, sure, but we're more like AW than WG is.

IMO it makes most sense to show you the *actual* minimum by default, rather than something that only *might* be the minimum. Yes, this will give Sonja/Flak weird damage numbers... and I think that's a good thing.
(You have correctly identified cart precedent, though, and that's also a fair argument)

Agreed, but unfortunately the annoying "vocal 5%" is part of the dev team in this case.
I strongly considered grabbing e.g. VIEWMODE to toggle between damage prediction types while previewing attack damage, but I didn't want to just throw it in willy-nilly without thinking through what I want the end-state of the controls to look like.